pontoon/administration/management/commands/sync_projects.py

```diff
@@ -0,0 +1,327 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from bulk_update.helper import bulk_update
+
+from pontoon.administration.files import update_from_repository
+from pontoon.administration.vcs import commit_to_vcs
+from pontoon.base.models import Entity, Locale, Project, Resource, Translation
+from pontoon.base.utils import search
+from pontoon.base.vcs_models import VCSProject
+
+
+class Command(BaseCommand):
+    args = '<project_id project_id ...>'
+    help = 'Synchronize database and remote repositories.'
+
+    def log(self, msg, *args, **kwargs):
+        """Log a message to the console."""
+        self.stdout.write(msg.format(*args, **kwargs) + u'\n')
+
```

Comment:
Is that newline character at the end even necessary? https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/